packages/lesswrong/components/common/TabNavigationMenu/menuTabs.tsx

```diff
@@ -124,6 +124,12 @@ export const menuTabs: Record<ForumTypeString,Array<MenuTab>> = {
       link: '/hpmor',
       tooltip: 'What if Harry Potter was a scientist? What would you do if the universe had magic in it? A story that illustrates many rationality concepts.',
       subItem: true,
+    }, {
+      id: 'bestoflesswrong',
+      title: 'Best of LessWrong',
+      link: '/bestoflesswrong',
+      tooltip: 'Top-voted posts from the 2018 and 2019 Review',
```

Comment:
I'd make it from "the annual reviews" so this doesn't accidentally become outdated next year.